### PR TITLE
Fix proxy cache warmup

### PR DESCRIPTION
**Pull request by jlabedo**

### CODE REVIEW

File: packages/Symfony/DepedencyInjection/EcotoneExtension.php [hunk 1]
```diff
@@ -36,7 +36,6 @@ public function load(array $configs, ContainerBuilder $container)
             ->withLoadCatalog($config['loadSrcNamespaces'] ? 'src' : '')
             ->withNamespaces($config['namespaces'])
             ->withSkippedModulePackageNames($skippedModules)
-            ->withCacheDirectoryPath($container->getParameter('kernel.cache_dir'))
```
Comment (jlabedo) [inline packages/Symfony/DepedencyInjection/EcotoneExtension.php:39]: @dgafka , is it ok to remove this ? The cache is configured with `ServiceCacheConfiguration` and will respect `%kernel.cache_dir%` *runtime* variable, so you can move your codebase around with its cache.

Comment (dgafka) [reply packages/Symfony/DepedencyInjection/EcotoneExtension.php:39]: Yea, that's cool. It's deprecated and to be dropped in new Ecotone 2.0, we use ServiceCacheConfiguration everywhere now.

**Comment by dgafka** [comment]
@jlabedo what was broken? The benchmarks went down now.

**Comment by jlabedo** [comment]
> @jlabedo what was broken? The benchmarks went down now.

The cache clear dont générate proxies.
The benchmarks are similar on my machine

**Comment by jlabedo** [comment]
@dgafka , I ran benchmarks again on CI, it is now down a bit.

File: Monorepo/CrossModuleTests/Tests/ProxyCacheGenerationTest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    private function execute(ConfiguredMessagingSystem $messagingSystem)
+    {
+        if (getenv('APP_ENV') !== 'prod') {
+            $this->markTestSkipped('Proxy cache must be warmed up only for prod environment');
+        }
+        $commandBusProxy = $messagingSystem->getCommandBus();
+        $reflection = new ReflectionClass($commandBusProxy);
+        $filename = $reflection->getFileName();
+        self::assertFalse(self::isEvalCode($filename), 'Proxy class should not be generated as eval code');
```
Comment (dgafka) [inline Monorepo/CrossModuleTests/Tests/ProxyCacheGenerationTest.php:42]: What does this check? The proxy class will not be valid php file?

**Review by dgafka — APPROVED** [review]

File: Monorepo/CrossModuleTests/Tests/ProxyCacheGenerationTest.php (hunk 2 above)
Comment (jlabedo) [reply Monorepo/CrossModuleTests/Tests/ProxyCacheGenerationTest.php:42]: When there is a proxy cache, filename will be: `/data/app/Monorepo/ExampleApp/Symfony/var/cache/prod/ecotone/ProxyManagerGeneratedProxy__PM__EcotoneModellingCommandBusGeneratedb0bae25e3a4d1b512de93c80e68c1b60.php`

When there is not it is:
`/data/app/vendor/friendsofphp/proxy-manager-lts/src/ProxyManager/GeneratorStrategy/EvaluatingGeneratorStrategy.php(54) : eval()'d code`

Comment (jlabedo) [reply Monorepo/CrossModuleTests/Tests/ProxyCacheGenerationTest.php:42]: Ouch, these tests are passing on my machine and not in CI. I have to dig into that...

**Pull request merged**